backend-rental/AppRental/Model/Rent.cs

```diff
@@ -0,0 +1,21 @@
+namespace AppRental.Model
+{
+    public class Rent
+    {
+        public int Id { get; set; }
+        public required Car Car { get; set; }
```

Comment:
Same as in `Offer.cs` -  we should be consistent and either use only Car or also include CarId